src/css/jwplayer/imports/video.less

```diff
@@ -9,6 +9,9 @@
         height: 100%;
         margin: auto;
         background: transparent;
+        -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+        -webkit-focus-ring-color: rgba(0, 0, 0, 0);
```

Comment:
`-webkit-focus-ring-color` Can be removed. It isn't a CSS property, but rather the macOS/Chrome vendor prefix for the default color's value.